packages/vite/src/node/plugins/css.ts

```diff
@@ -848,6 +851,7 @@ type StylePreprocessor = (
     [key: string]: any
     additionalData?: PreprocessorAdditionalData
     filename: string
+    alias: Alias[]
```

Comment:
**nitpick (non-blocking):** Plural of `alias` is `aliases` (applies also to other lines)

Reply:
Thank you for your advice. I've changed the name of `alias` to `aliases`.

Reply:
Sorry for going against this change, but we are using `alias` in other config options, and it is a string of aliases https://vitejs.dev/config/#resolve-alias. It is strange in the code to see `aliases = config.resolve.alias`. I think the singular form was ok here. @Shinigami92 @crcong what do you think?

Reply:
Mh I complained about it, because it was passed and always used as an array (in this PR)
In convention perspective this would mean that it should be named in plural form

Keep in mind that it is only a code change right now and not a breaking change, so `config.resolve.alias` is still `config.resolve.alias` in the public exposed API

Reply:
I found that the configuration `config.resolve.alias` has been processed into an array inside `vite`, so I also think it is ok to use the singular here. I will change it back. @Shinigami92 @matias-capeletto  Thank you for your suggestions.

Reply:
Okay, sorry for the inconvenience :sweat_smile: